code/controllers/subsystem/processing/flightpacks.dm

```diff
@@ -6,6 +6,22 @@ var/datum/controller/subsystem/processing/flightpacks/SSflightpacks
 	wait = 2
 	stat_tag = "FM"
 	flags = SS_NO_INIT|SS_TICKER|SS_KEEP_TIMING
+	var/flightsuit_processing = FLIGHTSUIT_PROCESSING_FULL
 
 /datum/controller/subsystem/processing/flightpacks/New()
 	NEW_SS_GLOBAL(SSflightpacks)
+
+/datum/controller/subsystem/processing/flightpacks/Initialize()
+	sync_flightsuit_processing()
+
+/datum/controller/subsystem/processing/flightpacks/vv_edit_var()
```

Comment:
This will runtime since it's missing arguments. Plus make it only update the processing on the correct variable change(in this case is the flightsuit_processing).